index.markdown

```diff
@@ -12,27 +12,16 @@ download_ctas:
 ctas:
     post: ' source on GitHub'
 
-headline: "Valkey makes it easy to find whatever you need in all of your data."
+headline: "Valkey: an open source, in-memory data store"
```

Comment:
I think we're slightly aligned to move more away from in-memory.

```suggestion
headline: "Valkey: an open source datastructure store"
```

Reply:
Isn't in-memory the #1 way in which Valkey is being used?  Shouldn't we always be trying to ensure that none of the changes to support non-ram-storage will ever cause a degradation of in-ram performance?  It seems like a good idea to advertise our area of highest performance.

Reply:
I don't disagree, but saying it's an "in-memory data store" short sells the other parts of it. Saying it's "primarily in-memory" also makes sense. I just think it's worth being opinionated about the differences.